R/redshift-transaction.R

```diff
@@ -43,11 +43,17 @@ transaction <- function(.data, .dbcon, .function_sequence) {
       message(e$message)
       DBI::dbExecute(.dbcon, "ROLLBACK;")
       message("Rollback complete")
-      FALSE
+      result <- FALSE
+      attr(result, "error") <- e
+      stop(glue("A redshift error occured: {e$message}"))
+      return(result)
```

Comment:
Doesn't the `stop()` mean that the `return()` is never reached?

Reply:
That's true.  There are a couple questionable decisions in that function that among them.  Might was well fix it though.